README.md

```diff
@@ -340,6 +340,16 @@ swagger_data = Swagger::Blocks.build_root_json(SWAGGERED_CLASSES)
 File.open('swagger.json', 'w') { |file| file.write(swagger_data.to_json) }
 ```
 
+#### Overriding attributes
+
+If certain attributes must be customized on-the-fly, (eg. a dynamic subdomain for the host) you can merge a hash containing the customized values on the returned JSON file. You can wrap ```build_root_json``` inside your own class:
```

Comment:
Three tiny changes:
- Can you remove `, (eg. a dynamic subdomain for the host)`
- `JSON.` instead of `JSON file.`
- `inside your own method` instead of `inside your own class`